ttrpc-codegen/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "ttrpc-codegen"
-version = "0.5.0"
-edition = "2018"
-authors = ["The AntFin Kata Team <kata@list.alibaba-inc.com>"]
-license = "Apache-2.0"
+version = "0.5.1"
```

Comment:
No need to bump, We can do this in a new PR which speciality updates ttrpc-codegen